tests/js/navButtonsTests.js

```diff
@@ -0,0 +1,85 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
+    gpii.tests.verifyStates = function (msg, backButton, nextButton, backDisabled, backVisible, nextDisabled, nextVisible) {
+        jqUnit[backDisabled? "assertTrue" : "assertFalse"](msg + " - The back button is disabled", backButton.is(":disabled"));
+        jqUnit[backVisible? "assertTrue" : "assertFalse"](msg + " - The back button is hidden", backButton.is(":visible"));
+        jqUnit[nextDisabled? "assertTrue" : "assertFalse"](msg + " - The next button is enabled", nextButton.is(":disabled"));
+        jqUnit[nextVisible? "assertTrue" : "assertFalse"](msg + " - The next button is shown", nextButton.is(":visible"));
+    };
+
+    gpii.tests.verifyButtons = function (that, currentPanelNum) {
+        jqUnit.assertEquals("The model value has been updated", currentPanelNum, that.model.currentPanelNum);
+
+        var msg,
+            start = that.options.panelStartNum,
+            end = that.options.panelTotalNum,
+            backButton = that.locate("back"),
+            nextButton = that.locate("next"),
+            backTooltip = that.backTooltip,
+            nextTooltip = that.nextTooltip;
+
+        if (currentPanelNum === start) {
+            msg = "On the start panel";
+            gpii.tests.verifyStates(msg, backButton, nextButton, true, false, false, true);
```

Comment:
this set of booleans is quite confusing to understand by looking at it. I think it would make more sense if it were an object.